src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -218,6 +218,16 @@ values of the log record limits
 * `OTEL_LOGRECORD_ATTRIBUTE_VALUE_LENGTH_LIMIT`
 * `OTEL_LOGRECORD_ATTRIBUTE_COUNT_LIMIT`
 
+## Environment Variables for Experimental Features
+
+### Otlp Log Exporter
+
+* `OTEL_DOTNET_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES`
```

Comment:
I think our convention for naming experimental environment variables should include the word `EXPERIMENTAL`.

`OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES `

Here's an example from java https://github.com/open-telemetry/opentelemetry-java/blob/5ccc7fccb3fd0bdfa902bfea7bcee43384cdcf3e/sdk-extensions/incubator/README.md?plain=1#L55